requirements.txt

```diff
@@ -6,6 +6,7 @@ isort==5.9.3
 lazy-object-proxy==1.6.0
 mccabe==0.6.1
 platformdirs==2.4.0
+psycopg2==2.8.6
```

Comment:
If anyone has issues with psycopg2 installation `pip install psycopg2-binary` worked for me.

Reply:
perhaps we can switch out `psycopg2` for `pscyopg2-binary`, so it doesn't have to be build locally as well

Reply:
I would disagree with having the binary package, mostly because of this: https://www.psycopg.org/docs/install.html#psycopg-vs-psycopg-binary 

Psycopg-binary bundles several libraries together which may be different from the system's one and this might trigger annoying issues at a certain point (if eventually the difference between the psycopg-binary and the system libraries "grows too much").

Reply:
Will keep it at the source package :)